src/main/resources/lib/folder/blockWrapper.jelly

```diff
@@ -0,0 +1,19 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:st="jelly:stapler">
+  <st:documentation>
+    Support both table and div layout, remove when baseline is 2.264 or newer
+    <st:attribute name="name" />
+  </st:documentation>
+  <j:choose>
+    <j:when test="${divBasedFormLayout}">
+      <div name="${attrs.name}">
+        <d:invokeBody/>
+      </div>
+    </j:when>
+    <j:otherwise>
+      <table name="${attrs.name}">
+        <d:invokeBody/>
+      </table>
+    </j:otherwise>
```

Comment:
Will this approach have the problem noted in https://github.com/jenkinsci/branch-api-plugin/pull/228, or is that problem specific to `branch-api`?

Reply:
possibly, it's the name attribute that isn't getting passed through.